### PR TITLE
Notifiarr to api

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
NO

#### Description
Pending Notifiarr changes

#### Todos
- Tests
- Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
- [Wiki Updates](https://wiki.servarr.com)
- Requires https://github.com/Readarr/Readarr/pull/1704

#### Issues Fixed or Closed by this PR

* Fixes #XXXX

### CODE REVIEW

**Review by austinwbest — DISMISSED** [review]

**Comment by bakerboy448** [comment]
good to go minus azure problems; unless @Qstick plans to land this in parallel with the webhook rewrite (please give Nit a heads up :D before that'd land)

**Pull request closed without merging**
